google-cloud-core/src/test/java/com/google/cloud/RoleTest.java

```diff
@@ -28,28 +28,31 @@
 
   @Test
   public void testOf() {
-    assertEquals("roles/viewer", VIEWER.getValue());
-    assertEquals("roles/editor", EDITOR.getValue());
-    assertEquals("roles/owner", OWNER.getValue());
+    assertThat(VIEWER.getValue()).isEqualTo("roles/viewer");
```

Comment:
Is the convention to have expected values first or second? In other code bases, I've seen (expected, actual). I'm fine following whatever is the convention here.

Reply:
Convention of `assertEquals` is to put expected value first. However, convention of `assertThat` is to [put actual first](https://google.github.io/truth/faq#order).

> That explanation is helpful. My comment was about including a link to an official reference to what you explain about "roles/owner" vs "projects/X/roles/Y", just for documentation purposes.

Ah I see. I can do that.